typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -179,6 +203,58 @@ export const ezEthSafes: Record<string, string> = {
   zircuit: '0x8410927C286A38883BC23721e640F31D3E3E79F8',
   taiko: '0x8410927C286A38883BC23721e640F31D3E3E79F8',
   sei: '0x0e60fd361fF5b90088e1782e6b21A7D177d462C5',
+  swell: '0x435E8c9652Da151292F3981bbf663EBEB6668501',
+};
+
+const existingProxyAdmins: ChainMap<{ address: string; owner: string }> = {
```

Comment:
was this necessary to add?

We only needed the proxy admins explicitly set for the zerion warp routes because we were temporarily taking ownership of them but wanted to be able to easily transfer them to the zerion team. Because here Renzo owns everything themselves, we shouldn't need to specify the proxy admins explicitly

Reply:
Removed